pkg/crc/network/nameservers.go

```diff
@@ -33,7 +33,7 @@ func GetResolvValuesFromInstance(sshRunner *ssh.Runner) (*ResolvFileValues, erro
 	return parseResolveConfFile(out)
 }
 
-func CreateResolvFileOnInstance(sshRunner *ssh.Runner, resolvFileValues ResolvFileValues) error {
+func UpdateResolvFileOnInstance(sshRunner *ssh.Runner, resolvFileValues ResolvFileValues) error {
```

Comment:
One more instance of `resolve.conf` in the commit log.

And maybe split the short log, it's very long
```
nameserver: rename CreateResolvFileOnInstance

It's renamed to UpdateResolvFileOnInstance. The resolv.conf file is always present in the VM, this function ...
```

(I disagree that `cp myresolv.conf /etc/resolv.conf` is an "update", but that's a different matter, not terribly important here ;)
```

Reply:
done.